tcms/testruns/tests/test_permissions.py

```diff
@@ -128,3 +128,43 @@ def verify_post_with_permission(self):
             self.assertEqual(case.history.latest().history_id, case_run.case_text_version)
             self.assertEqual(last_run.build, case_run.build)
             self.assertEqual(None, case_run.close_date)
+
+
+class MenuAddCommentItemTestCase(PermissionsTestCase):
+    permission_label = 'django_comments.add_comment'
+    http_method_names = ['get']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.product = factories.ProductFactory(name="Kiwi")
+        cls.plan = factories.TestPlanFactory(product=cls.product)
```

Comment:
you don't really need these other attributes. TestRunFactory will create them automatically and when asserting if you are on the correct page you can read the information from the `self.test_run` attribute and its attributes.